group-coordinator/src/test/java/org/apache/kafka/coordinator/group/OffsetMetadataManagerTest.java

```diff
@@ -181,10 +181,28 @@ public List<OffsetFetchResponseData.OffsetFetchResponseTopics> fetchOffsets(
             String groupId,
             List<OffsetFetchRequestData.OffsetFetchRequestTopics> topics,
             long committedOffset
+        ) {
+            return fetchOffsets(
+                groupId,
+                "",
+                -1,
+                topics,
+                committedOffset
+            );
+        }
+
+        public List<OffsetFetchResponseData.OffsetFetchResponseTopics> fetchOffsets(
+            String groupId,
+            String memberId,
+            int memberEpoch,
+            List<OffsetFetchRequestData.OffsetFetchRequestTopics> topics,
+            long committedOffset
         ) {
             OffsetFetchResponseData.OffsetFetchResponseGroup response = offsetMetadataManager.fetchOffsets(
                 new OffsetFetchRequestData.OffsetFetchRequestGroup()
                     .setGroupId(groupId)
+                    .setMemberId(memberId)
```

Comment:
Is this needed now because of validation (and wasn't needed before)

Reply:
Correct.